playbook

```diff
@@ -0,0 +1,65 @@
+---
+- name: Install To Do app on server
+  hosts: managed
+  remote_user: ec2-user
+  vars_prompt:
+
+  - name: trello_key
+    prompt: Trello key
+    private: false
+
+  - name: trello_token
+    prompt: Trello token
+    private: false
+
+  tasks:
+  - name: Install Git
+    become: yes
+    ansible.builtin.yum:
+      name: git
+      state: present
+  
+  - name: Install Python
+    become: yes
+    ansible.builtin.yum:
+      name: python3
+      state: present
```

Comment:
Worth noting that these can be combined like so:

```suggestion
  - name: Install Git & Python
    become: yes
    ansible.builtin.yum:
      name: 
        - git
        - python3
      state: present
```

Up to you whether you think that's easier or harder to read though

Reply:
It is, but I think I prefer the individual steps.